recipes/css-selector.yaml

```diff
@@ -0,0 +1,15 @@
+data:
+  - title
+  - short_title?
+  - mdn_url
+body:
+  - prose.short_description
+  - prose.syntax
```

Comment:
It occurs to me now that `prose.syntax` could be more structured too (`data.syntax` could be a section + a code block, rather than arbitrary prose)

Reply:
This is a can of worms. For the JS docs we could also have chosen to structure `prose.syntax`. In method pages this might things like parameters and return value as well as a code block (see e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#Syntax). In 

We decided that although we might want to add this structure in the future, for the time being we should just treat it as unstructured prose.

I think I'd prefer us to be consistent in this decision. Partly this is because the details of the structure might be different from one page type to another (for example, CSS selectors might just have code block+prose, while JS methods might have code block+parameters+returns). And it'll be easier to sort all this our when we have a complete set of `prose.syntax` examples to consider.

Also there's naming the ingredient. If the syntax structure for JS methods is different from that for CSS selectors then that's two different ingredients, which should have two different names. So we shouldn't call this `data.syntax`. We could perhaps have two syntax ingredients, `data.method_syntax` that includes parameters and return values, and `data.simple_syntax` or something better, that's just a code block and some optional prose. But this seems like the kind of conversation that's easier to have when we're looking at all the page types (including Web API and HTTP and the rest).

Reply:
OK, let's not open this particular can just yet. I agree that it makes sense to put this off until we've looked at other page types.